styles.css

```diff
@@ -91,6 +95,331 @@ ul li a {
+
+.card-1-ul,
+.card-2-ul,
+.card-3-ul,
+.card-4-ul,
+.card-5-ul,
+.card-6-ul {
+  display: flex;
+  gap: 12px;
+  padding: 5px 24px;
+}
+
+.card-1 .html-btn,
+.card-2 .html-btn,
+.card-3 .html-btn,
+.card-4 .html-btn,
+.card-5 .html-btn,
+.card-6 .html-btn {
+  border: 1px solid #8993a4;
+  width: 56px;
+  height: 40px;
+  font-weight: 500;
+  font-family: arial, sans-serif, 'Inter';
+  font-size: 15px;
+  line-height: 20px;
+  color: #091e42;
+  cursor: pointer;
+}
+
+.card-1 .bs-btn,
+.card-2 .bs-btn,
+.card-3 .bs-btn,
+.card-4 .bs-btn,
+.card-5 .bs-btn,
+.card-6 .bs-btn {
+  border: 1px solid #8993a4;
+  width: 95px;
+  height: 40px;
+  font-weight: 500;
+  font-family: arial, sans-serif, 'Inter';
+  font-size: 15px;
+  line-height: 20px;
+  color: #091e42;
+  cursor: pointer;
+}
+
+.card-1 .ruby-btn,
+.card-2 .ruby-btn,
+.card-3 .ruby-btn,
+.card-4 .ruby-btn,
+.card-5 .ruby-btn,
+.card-6 .ruby-btn {
+  border: 1px solid #8993a4;
+  width: 61px;
+  height: 40px;
+  font-weight: 500;
+  font-family: arial, sans-serif, 'Inter';
+  font-size: 15px;
+  line-height: 20px;
+  color: #091e42;
+  cursor: pointer;
+}
+
+.card-1 .see-pro,
+.card-2 .see-pro,
+.card-3 .see-pro,
+.card-4 .see-pro,
+.card-5 .see-pro,
+.card-6 .see-pro {
+  width: 100%;
+  height: 48px;
+  background-color: #ff6b00;
+  margin-top: 10px;
+  border: none;
+  cursor: pointer;
+}
+
+.about-me {
+  margin: 0 5px;
+  padding: 0;
+  background-color: #1c1a19;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.basic-info {
+  display: flex;
+  flex-direction: column;
+  align-items: start;
+  padding-top: 100px;
+  background-image: url("./Img/Group.svg");
+  background-repeat: no-repeat;
+  background-position: bottom;
+  background-position-x: 100%;
+  background-position-y: 90%;
+}
+
+.basic-info h2 {
+  font-size: 72px;
+  font-weight: 400;
+  font-family: arial, sans-serif, 'Crete Round';
+  line-height: 88px;
+  color: #ff6b00;
+  padding: 0 24px;
+}
+
+.basic-info p {
+  font-size: 16px;
+  color: #f4f5f7;
+  font-weight: 400;
+  font-family: arial, sans-serif, 'Inter';
+  line-height: 24px;
+  padding: 0 24px;
+}
+
+.basic-info .rus-btn {
+  width: 161px;
+  height: 48px;
+  background-color: #ff6b00;
+  border: none;
+  cursor: pointer;
+}
+
+.basic-info ul {
+  padding: 24px 0 100px 24px;
+}
+
+.btm-bor {
+  padding: 24px;
+}
+
+.lan-tag,
+.fw-tag,
+.skill-tag {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.rect-55 {
+  width: 12px;
+}
+
+.rect-56 {
+  width: 8px;
+}
+
+.rect-57 {
+  width: 10px;
+}
+
+.language h2,
+.freamworks h2,
+.skills h2 {
+  color: #fff;
+  font-family: 'InaiMathi', arial, sans-serif;
+  font-size: 26px;
+  font-weight: 700;
+  text-align: center;
+  padding: 20px 0;
+}
+
+.language ul,
+.freamworks ul {
+  padding-left: 40px;
+}
+
+.language ul li,
+.freamworks ul li,
+.skills ul li {
+  padding: 10px 0;
+  font-family: arial, sans-serif, 'Inter';
+  font-weight: 400;
+  font-size: 16px;
 }
```

Comment:
- [ ] Please 🙏🏽 add a space between the bullet points and the title and a space at the end to be similar to the Figma template 👇🏽 

| Figma template | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/219765906-383146ab-7e0b-4787-b74c-63f14b7c65c2.png) | ![image](https://user-images.githubusercontent.com/58642949/219765981-344a6314-1120-4966-97cb-c8a33ceee624.png) |